tests/unittest/test_helpers.py

```diff
@@ -8,5 +12,162 @@ def test_dummy(self):
         pass
 
 
+class TestProperties(UnittestWithTmpFolder):
```

Comment:
That's very good we already have tests

Reply:
Thanks Enrico! Felt this was necessary especially after I got burnt during the demo!